File: Demo/temp/temp/ViewController.m
//
//  ViewController.m
//  temp
//
//  Created by JustinLau on 2019/7/5.
//  Copyright © 2019 JustinLau. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    NSDictionary *vipInfo = @{
                              @"type": @(1),
                              @"data": @{}
                              };
    NSDictionary *hehe = @{
                           @"gameInfo": vipInfo
                           };
    NSData *jsonedValue = [NSJSONSerialization dataWithJSONObject:hehe options:0 error:nil];
    NSString *jsonText = [[NSString alloc] initWithData:jsonedValue encoding:NSUTF8StringEncoding];
    NSLog(@"%@", jsonText);
}


@end

File: Code/2048Self/2048Self/ViewController.swift
//
//  ViewController.swift
//  2048Self
//
//  Created by JustinLau on 2019/7/29.
//  Copyright © 2019 JustinLau. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        let game = GameBoardViewController()
    }

}

